cpp/src/Slice/Parser.cpp

```diff
@@ -660,52 +661,38 @@ namespace
                     comment.erase(pos, endpos - pos + 1);
                 }
             } while (pos != string::npos);
+        }
 
-            const string link = "{@link";
-            pos = 0;
-            do
+        // Fix any javadoc using the provided link formatter.
+        const string link = "{@link ";
```

Comment:
There should always be a space after the `@link`.
This code would let through things like `{@linkStringSeq}` which is bogus.

Reply:
`@link` cannot span multiple lines, so `\n` shouldn't be a problem.
But, you're right that `\t` used to work and won't anymore. But I'm pretty sure it only worked by accident.
Using `\t` here would be putting tabs between the words in a sentence. It would work, but it's a very strange thing to do. I'm not convinced any sane person is doing this.